Unix/wsman/wsmanparser.c

```diff
@@ -3241,29 +3240,66 @@ int WS_ParseFaultBody(
                 }
                 else if (Tcscmp(e.data.data, ZT("WSManFault")) == 0)
                 {
+                    /* fault->mi_result should be MI_RESULT_FAILED for WSManFault */
+                    fault->mi_result = MI_RESULT_FAILED;
```

Comment:
Ah, so here is the catch here! The fault result code needs to get matched to the most appropriate error code for the fault. There is some code somewhere for this. I will take a look in a while to see if I can track it down. Otherwise a TIMEOUT error will not result in a MI_RESULT_TIMEOUT error.